styles/style.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=BhuTuka+Expanded+One&family=Roboto:wght@400;500&display=swap');
```

Comment:
this way doesn't work, you should add the link to `<link>` tag in html
if you want to add a font to css, you should download the font